### PR TITLE
Update version string for all packages

### PR DESCRIPTION
**Pull request by wjxiz1992**
- Update cudf version from 0.14 to 0.15
- Update XGBoost version from 0.1.0 to 0.2.0
- Update rapids plugin version from 0.1.0 to 0.2.0

### CODE REVIEW

File: examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb [hunk 1]
```diff
@@ -147,7 +147,7 @@
     "```scala\n",
     "import org.apache.spark.sql.SparkSession\n",
     "val spark = SparkSession.builder().appName(\"Taxi-GPU\").getOrCreate\n",
-    "%AddJar file:/data/libs/cudf-0.14-SNAPSHOT-cuda10.jar\n",
+    "%AddJar file:/data/libs/cudf-0.15-SNAPSHOT-cuda10.jar\n",
```
Comment (GaryShen2008) [inline examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: Should we update all the versions here? cudf/xgboost4j/rapids-4-spark.

Comment (wjxiz1992) [reply examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: Updated the rest versions

Comment (GaryShen2008) [reply examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: Why we need the SNAPSHOT string?

Comment (wjxiz1992) [reply examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: It seems to be a "tech debt", they are removed now.

**Review by GaryShen2008 — APPROVED** [review]

**Pull request merged**